comfy-mobile-ui-api-extension/utils/global_websocket_manager.py

```
@@ -68,9 +68,102 @@ def __init__(self):
+                size = getattr(value, "size", None)
+                if isinstance(size, tuple):
+                    size = list(size)
+                return {
+                    "__type__": "image",
+                    "class": value_type_name,
+                    "mode": mode,
+                    "size": size
+                }
+
+            # Numpy-like arrays/scalars if available.
+            if hasattr(value, "tolist") and callable(getattr(value, "tolist")):
```

Ruff (0.15.5)

[warning] Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)